Cargo.toml

```diff
@@ -30,7 +31,7 @@ unicode-segmentation = "1.2"
 unicode-width = "0.1"
 termion = { version = "1.5", optional = true }
 rustbox = { version = "0.11", optional = true }
-crossterm = { version = "0.6", optional = true }
+crossterm = { optional = true, git = "https://github.com/TimonPost/crossterm", branch = "fixes" }
```

Comment:
I assume that it is meant to be `{ version = "0.9", optional = true }`

Reply:
No, currently I have to do a release with the changes on this branch, it has a couple of fixes, also useful for TUI. I think that release could be done this week when it is done, I will update this PR with the new version.